dts/common/nordic/nrf54l15.dtsi

```diff
@@ -577,6 +578,13 @@
 				interrupts = <270 NRF_DEFAULT_IRQ_PRIORITY>;
 				status = "disabled";
 			};
+
+			reg: regulator@120600 {
+				compatible = "nordic,nrf5x-regulator";
+				reg = <0x120600 0x1>;
+				regulator-name = "REG";
+				regulator-initial-mode = <NRF5X_REG_MODE_DCDC>;
+			};
```

Comment:
this seems wrong: there are 2 regulators that are children of this IP block: VREGM, and VREGMAIN

Reply:
As adviced: Added group for regulators, new dts description for nrf54x regulators, commits squashed